pom.xml

```diff
@@ -1095,6 +1095,7 @@
 
             <!-- Used in internal utilities (protected by conditional) -->
             <ignore>java.lang.Long</ignore>
+            <ignore>java.lang.Character</ignore>
```

Comment:
hmmm... not really... Why not just always use what you have here:
https://github.com/netty/netty/pull/4603/files#diff-59b216d6f5eb79a60f2b338e050c259eR395